django-stubs/db/models/functions/comparison.pyi

```diff
@@ -8,6 +8,7 @@ class Cast(Func):
     def __init__(self, expression: Any, output_field: Union[str, Field]) -> None: ...
 
 class Coalesce(Func): ...
+class Collate(Func): ...
```

Comment:
can you type out its `__init__` please? it takes an expression, and a collation that's a string, it seems

Reply:
sure, i have updated the MR